pkg/cloudprovider/gce/gce.go

```diff
@@ -154,6 +178,7 @@ func newGCECloud(config io.Reader) (*GCECloud, error) {
 		projectID:        projectID,
 		zone:             zone,
 		instanceID:       instanceID,
+		networkName:      networkName,
```

Comment:
Which component is calling this? We can't assume the nodes are on the same network as the master.